src/OrientationAPI/WebApplication1/DataAccess/CustomerDataAccess.cs

```diff
@@ -49,26 +49,26 @@ public bool InactivateCustomer(int entitytoUpdate)
 
             }
         }
-          
-			public void Update(CustomerListResult customer)
-			{
-				using (var connection =
-				new SqlConnection(ConfigurationManager.ConnectionStrings["Bangazon"].ConnectionString))
-				{
-					connection.Open();
-					var result = connection.Execute("Update Customer " +
-																	"Set FirstName = @firstname, LastName = @lastname " +
-																	"Where CustomerId = @customerId", new { FirstName = customer.FirstName, LastName = customer.LastName, CustomerId = customer.CustomerId });
-					return;
-				}
-			}
 
+        public void Update(CustomerListResult customer)
+        {
+            using (var connection =
+            new SqlConnection(ConfigurationManager.ConnectionStrings["Bangazon"].ConnectionString))
+            {
+                connection.Open();
+                var result = connection.Execute("Update Customer " +
+                                                                "Set FirstName = @firstname, LastName = @lastname " +
+                                                                "Where CustomerId = @customerId", new { FirstName = customer.FirstName, LastName = customer.LastName, CustomerId = customer.CustomerId });
+                return;
+            }
         }
 
+    }
 
     public interface IRepository<T>
     {
         List<T> GetAll();
         bool InactivateCustomer(int entityToUpdate);
-	}
+        void Update(T customer);
```

Comment:
The name of this parameter indicates that the implementation should be unique to a customer repository. You should make this parameter name more generic. Such as "entityToUpdate".